aquarius/app/assets.py

```diff
@@ -262,6 +263,10 @@ def register():
     for service in _record['service']:
         if service['type'] == 'Metadata':
             service_id = int(service['serviceDefinitionId'])
+            if Config(filename=app.config['CONFIG_FILE']).allow_free_assets_only == 'true':
+                if _record['service'][service_id]['metadata']['base']['price'] != 0:
```

Comment:
I thought price was a string now, or is it converted to an integer internally?

Reply:
good point